c7n/resources/backup.py

```diff
@@ -89,3 +93,62 @@ def get_resources(self, resource_ids, cache=True):
 class KmsFilter(KmsRelatedFilter):
 
     RelatedIdsExpression = 'EncryptionKeyArn'
+
+
+@BackupVault.action_registry.register('modify-policy')
+class ModifyPolicyStatement(ModifyPolicyBase):
+    """Action to modify Backup Vault IAM policy statements.
+
+    :example:
+
+    .. code-block:: yaml
+
+           policies:
+              - name: backup-vault-get-policy
+                resource: backup-vault
+                filters:
+                  - type: value
+                    key: BackupVaultName
+                    value: "backup_vault_name"
+                actions:
+                  - type: modify-policy
+                    add-statements:
+                      - "Sid": "AddMe"
+                        "Effect": "Allow"
+                        "Principal": {"AWS": "arn:aws:iam::116249476610:root"}
+                        "Action": ["backup:GetBackupVaultAccessPolicy"]
+                        "Resource": "*"
+                    remove-statements: []
+    """
+    permissions = ('backup:PutBackupVaultAccessPolicy', 'backup:GetBackupVaultAccessPolicy')
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('backup')
+
+        for r in resources:
+            try:
+                policy = json.loads(
```

Comment:
do backup plans allow cross-account grants in their iam policies, if so we also want a cross-account filter. which would typically annotate the resource with the plan such that the policy should only be feteched if CrossAccountAccessFilter.annotation_key isn't present